apps/wallet-mobile/src/features/Swap/common/AmountCard/AmountCard.tsx

```diff
@@ -39,8 +39,6 @@ export const AmountCard = ({
 }: Props) => {
   const strings = useStrings()
   const {quantity, tokenId} = amount
-  const amountInputRef = useRef<TextInput>(inputRef?.current ?? null)
```

Comment:
👀

Reply:
I don't know, I think that worked before, passing the parent's ref to useRef to reuse it. Could be something to do with it coming from a context now. Or maybe never worked like that.